docs/flip-flop_conversions.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: default
+title: Flip-Flop Conversions
+comments: true
+nav_order: 14
+---
+
+# Flip-Flop Conversions
+{: .no_toc }
+
+This page will teach you how to convert one type of Flip flop to another type.
+
+<h2>Steps to be followed :</h2>
+ <ul>
+   <li> Identify available and required Flip flop. </li>
+   <li> Make characteristic table for required Flip flop. </li>
+   <li> Make excitation table for available Flip flop. </li>
+   <li> Write boolean expression for available Flip flop. </li>
+   <li> Draw the circuit. </li>
+ </ul>
```

Comment:
Change this to an ordered list and remove heading and trailing spaces. Use markdown syntax.

E.g.
```
1. Identify...
2. Make...
```